git-rebase-all

```diff
@@ -45,7 +45,7 @@ if [ $# -eq 2 ]; then
   root=$2
 
   # Fetch all the branches that contain the root of the subtree
-  branches=$(git branch --contains $root | cut -c 3-)
+  branches=$(git branch --contains $root --no-merged $new_upstream | grep -v '* (HEAD' | cut -c 3-)
```

Comment:
Make this `* (HEAD detached` so it's clear what it's filtering out? I wonder if there's a more flexible command for listing branches... `git for-each-ref refs/heads` perhaps?